backend/src/main/cards/playable_card_logic.py

```diff
@@ -0,0 +1,48 @@
+
+class PlayableCardLogic:
+    def __init__(self, hand):
+        self.hand = hand
```

Comment:
Shouldn't `hand`, `discard`, and `lost` all be of type `Cards()` rather than just a list?